### PR TITLE
CN-4874 Create HODLViewCode Screen

### PR DESCRIPTION
**Pull request by ShamanShaulic**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/screens/HodlDeactivate/HodlDeactivate.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import CelButton from "../../atoms/CelButton/CelButton";
+
+@connect(
+  state => ({
+    theme: state.user.appSettings.theme,
+    formData: state.forms.formData,
+  }),
+  dispatch => ({ actions: bindActionCreators(appActions, dispatch) })
+)
+class HodlDeactivate extends Component {
```
Comment (fjovakaric) [inline app/components/screens/HodlDeactivate/HodlDeactivate.js]: NAme it HODLViewCode, also rename file and folder

Comment (fjovakaric) [reply app/components/screens/HodlDeactivate/HodlDeactivate.js]: Rename PR and change the task number. This is CN-4874
